website/docs/guides/message-extraction.md

```diff
@@ -2,7 +2,7 @@
 
 Message extraction is an essential step in the internationalization process. It involves analyzing your code and extracting all messages defined in it so that your message catalogs are always up-to-date with the source code.
 
-To extract messages from your application with the lingui functions, use the `lingui extract` command.
+To extract messages from your application with the Lingui cli, use the `lingui extract` command.
```

Comment:
i meant here that `lingui extract` command is extracting messages from lingui functions usages `t`, `<Trans>` etc